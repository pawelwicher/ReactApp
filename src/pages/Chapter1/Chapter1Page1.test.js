import React from 'react';
import Chapter1Page1 from './Chapter1Page1';
import renderer from 'react-test-renderer';

it(' Chapter1Page1 renders correctly', () => {
  const tree = renderer
    .create(<Chapter1Page1 />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});