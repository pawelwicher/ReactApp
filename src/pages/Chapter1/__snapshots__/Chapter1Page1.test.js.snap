// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Chapter1Page1 renders correctly 1`] = `
Array [
  <h1
    className="display-4"
  >
    Chapter 1 - Page 1
  </h1>,
  <p
    className="lead"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Diam quam nulla porttitor massa. Sit amet aliquam id diam maecenas ultricies mi eget. In nisl nisi scelerisque eu ultrices vitae auctor eu augue. Neque sodales ut etiam sit amet nisl.
  </p>,
  <br />,
  <pre>
    <code>
      
let i = 1
let a = (1, 2)
let xs = [1; 2; 3]
    
    </code>
  </pre>,
]
`;
